### ISSUE TITLE
notification when specific users log in

### ISSUE THREAD

**Issue by justdave** [justdave/MCStatus#35]
Suggested by Richard Edwards in a Play Store review

Would require a backgroundn service to poll the servers

**Comment by JLeif** [comment]
Hi Dave,
I'm new to Github, an unfortunately don't know Java or Python ( I wish I did ) I'm more of a backend Web Dev, Php and Mysql...
I just started a course on Javascript ( I use it in dev but, want to be a Lot better ).

Anyways, Are you considering adding a Phone Push Notification when a user logs in? This is the one thing i've wanted. I know only a little about Phone things... But I'm assuming you'd need to make MCstatus a background service and also would need to add a auto refresh to get new data from the server(s). Any further thoughts on the subject?

Thanks,
Justin

**Comment by justdave** [comment]
Yes, that's exactly what would need to happen.  Well, sorta.  Google doesn't allow background services anymore, it would have to be a service worker with a periodic alarm to trigger it, but it's the same general idea.
